templates/travis/.travis/performance_test_script.sh.j2

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+{% include 'header.j2' %}
+
+set -euv
+
+django-admin runserver 24817 >> ~/django_runserver.log 2>&1 &
```

Comment:
This won't work because pulp-api is in a container.

Reply:
I copied it from here: https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/publish_client_pypi.sh.j2#L9